en-US/mozorg/internet-health.lang

```diff
@@ -0,0 +1,110 @@
+## Note: Demo server is available at https://www-dev.allizom.org/internet-health
+## URL: https://www-dev.allizom.org/%LOCALE%/internet-health
+
+
+;Keeping the Internet healthy
+Keeping the Internet healthy
+
+
+;The Internet is our largest shared global resource.
+The Internet is our largest shared global resource.
```

Comment:
Bah, we need to fix this.
* Add note: `Used as HTML description of the page`
* Expose this string as: `The Internet is our largest shared global resource. Help us keep it healthy, open and safe.` and remove the following string, I'm going to open a PR to fix it.